src/main/java/app/PostController.java

```diff
@@ -0,0 +1,90 @@
+package app;
+
+
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+@RestController
+public class PostController {
+
+    @GetMapping("/")
+    public String elo() {
+        return "ELO";
+    }
+
+
+    @GetMapping(path = "/api/posts")
+    public void getPosts() {
+
+        RestTemplate restTemplate = new RestTemplate();
+
+        ResponseEntity<PostDto[]> responseEntity = restTemplate.exchange(
+                "https://jsonplaceholder.typicode.com/posts",
```

Comment:
read about properties in spring:
https://www.baeldung.com/properties-with-spring

move urls to properties